policies/release-requirements.md

```diff
@@ -0,0 +1,138 @@
+  that the source is ready for a release with a vote.
+
+Major and Minor Releases
+------------------------
+
+As the release comes after the beta releases there is no need to repeat the
+stability requirements as those should be held already by the beta releases.
+
+- All issues [triaged] as regressions on the development branch from which the
+  release is to be done must have a milestone assigned.
+  _Regressions are considered from the previous stable release series._
+- All issues or pull requests with the milestone for the release are closed.
+- There are no outstanding untriaged Coverity issues.
+- Coveralls coverage has not decreased overall from the previous release from
+  the particular development branch.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen for at least 7 days. No changes apart from regression
+  or security fixes should be merged during the freeze.
+- For 2 days before the release there should be no changes to ensure the daily
+  CI builds run on the development tree tip.
+- The OTC should explicitly approve that the source is ready for a release with
+  a vote.
+
+Patch Releases
+--------------
+
+The patch releases follow a similar process to major and minor releases with
+some simplifications as they are much more frequent and the tree stability
+requirements for the stable development branches should ensure there is
+a minimum amount of regressions in between the patch releases.
+
+- All issues [triaged] as regressions on the development branch from which the
+  release is to be done must have a milestone assigned.
+  _Regressions are considered from the previous minor, major or patch release
+  from the development branch._
+- All issues or pull requests with the milestone for the release are closed.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen for at least 3 business days. No changes apart from
+  regression or security fixes should be merged during the freeze.
+- For 2 days before the release there should be no changes to ensure the daily
+  CI builds run on the development tree tip.
+- Embargoed security fixes are excepted from the rule above as they cannot
+  be merged to the public tree before the release is being prepared.
+
+Triage Process
+--------------
+
+The issues and pull requests in GitHub must be assigned a `triaged: *` label by
+an OTC member according to what is the type (feature, bug, documentation,
+refactoring, ...) of the issue or pull request. When the triage happens for an
+issue or pull request that is a bug/bug fix, it must be assessed whether the
+bug is a regression or not.
+
+In general regressions should be fixed as soon as possible, optimally before
+the next release from the development tree is done. However sometimes that
```

Comment:
It's unclear to me what you mean with development tree. I suggest you make it: "before the next release is done"

Reply:
Too late. The vote has started.

Reply:
You can always create an other PR to fix that.